### DISCUSSION TITLE
post: Framer Motion Examples

### DISCUSSION THREAD

**Discussion by giscus** [wghglory/my-website#54] [Announcements]
# post: Framer Motion Examples

Guanghui Wang Website

https://derekw.vercel.app/posts/framer-motion

**Comment by jimwilson94** [comment]
How about the Vue ecosystem?

**Reply by wghglory** [reply]
1. You can check out https://github.com/vueuse/motion, which is heavily inspired by framer-motion. 
2. Some people suggest https://popmotion.io/pose/learn/vue-get-started/ although I'm not familiar with it.
3. Leverage framework-agnostic lib like GSAP.

Hope these references would be helpful.

**Reply by jimwilson94** [reply]
Thank you @wghglory
